windows/src/engine/mcompile/mc_unicode.cpp

```diff
@@ -261,11 +261,11 @@ const WCHAR cp1252[256] = {
 void InplaceUnicode(PWSTR p) {
   while(p && *p) {
     if(*p != UC_SENTINEL) {
-      if(*p >= 0 && *p <= 0xFF) {
```

Comment:
Was the `>=0` check redundant here?  Otherwise, why the change?

Reply:
`PWSTR` is a pointer to unsigned words, so `>= 0` is always true. Coverity picks it up because it suggests that we might have a logic problem.